features/support/electron.js

```diff
@@ -85,6 +85,13 @@ defineSupportCode(({ BeforeAll, Before, After, AfterAll, setDefaultTimeout }) =>
     });
   });
 
+  // this ensures that the spectron instance of the app restarts
+  // after the node update acceptance test shuts it down via 'kill-process'
+  // eslint-disable-next-line prefer-arrow-callback
+  After({ tags: '@preventAppQuit' }, async function () {
```

Comment:
maybe `@preventAppQuit` should be renamed to `@restartApp`

Reply:
Your request has been processed!